tests/model_serving/maas_billing/maas_api_key/utils.py

```
@@ -244,6 +244,43 @@ def build_inference_url(maas_scheme: str, maas_host: str, model_name: str) -> st
+    """Poll until an AuthPolicy exists and its Accepted condition is True."""
+    auth_policy = AuthPolicy(
+        client=admin_client,
+        name=policy_name,
+        namespace=namespace,
+    )
+    try:
+        for _ in TimeoutSampler(
+            wait_timeout=timeout,
+            sleep=5,
+            func=auth_policy.get,
+            exceptions_dict={NotFoundError: [], ResourceNotFoundError: []},
```

Flake8 (7.3.0)

[error] undefined name 'NotFoundError'

(F821)